cmd/zc_traverser_blob.go

```diff
@@ -76,14 +76,7 @@ func (t *blobTraverser) traverse(processor objectProcessor, filters []objectFilt
 	// check if the url points to a single blob
 	blobProperties, isBlob := t.getPropertiesIfSingleBlob()
 	if isBlob {
-		storedObject := newStoredObject(
-			getObjectNameOnly(blobUrlParts.BlobName),
-			"", // We already know the object's location -- no need.
-			blobProperties.LastModified(),
-			blobProperties.ContentLength(),
-			blobProperties.ContentMD5(),
-			blobProperties.BlobType(),
-			"") // We already know the container -- no need.
+		storedObject := newStoredObject(getObjectNameOnly(blobUrlParts.BlobName), "", blobProperties.LastModified(), blobProperties.ContentLength(), blobProperties.ContentMD5(), blobProperties.BlobType())
```

Comment:
Minor: Why the formatting change? The formatting change to a single line makes it harder to read. (And it makes the diffs harder to read.  I'd suggest that generally, when actioning PR comments, best to avoid unnecessary format changes, since any format change makes the diff harder for reviewers to read.  And if fact it will also adversely affect all historical diffs (version-to-version) and use of annotate/blame.  Basically, unnecessary formatting changes make it harder to read code history, since they make diff tools overstate the extent of the change.)

Reply:
Oh, it was just accidental. IDE didn't retain formatting when I auto-refactored

Reply:
Ew, that was nasty on the part of the IDE.

Reply:
Resolved.